quick-start.sh

```diff
@@ -38,6 +38,9 @@ if [ -z "$PROJECT_ID" ]; then
     exit 1
 fi
 
+# Enable APIs needed for Terraform
+gcloud services enable serviceusage.googleapis.com cloudresourcemanager.googleapis.com
```

Comment:
Ran into this issue: https://stackoverflow.com/a/72094901/5365414

Reply:
Can you please share the conversation we had around this with @bryan-skylight because it will be important for his work on the batch version of this script?